modules/core/02-client/client/cli/tx.go

```diff
@@ -241,6 +244,62 @@ func NewUpgradeClientCmd() *cobra.Command {
 	return cmd
 }
 
+// NewCmdRecoverClientProposal defines the command to recover an IBC light client
+func NewCmdRecoverClientProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "recover [subject-client-id] [substitute-client-id] [flags]",
+		Args:  cobra.ExactArgs(2),
+		Short: "recover an IBC client",
+		Long: "Submit a recover IBC client proposal along with an initial deposit.\n" +
+			"Please specify a subject client identifier you want to recover..\n" +
+			"Please specify the substitute client the subject client will be recovered to.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			proposal, err := govcli.ReadGovPropFlags(clientCtx, cmd.Flags())
+			if err != nil {
+				return err
+			}
+
+			subjectClientID := args[0]
+			substituteClientID := args[1]
+
+			authority, _ := cmd.Flags().GetString(FlagAuthority)
+			if authority != "" {
+				if _, err = sdk.AccAddressFromBech32(authority); err != nil {
+					return fmt.Errorf("invalid authority address: %w", err)
+				}
+			} else {
+				authority = sdk.AccAddress(address.Module("gov")).String()
```

Comment:
this is what the sdk is using for their `tx` cli.

From what I've understood, this is equivalent to:

```
authtypes.NewModuleAddress(govtypes.ModuleName).String()
```

if people have a preference, I can switch it over.

Reply:
Either is fine by me, but would be nice not to hardcode a string and instead reference `govtypes.ModuleName`

Reply:
yup, solid point